docs/book/features/vsphere_csi_vmc.md

```diff
@@ -0,0 +1,85 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - VMware Cloud on AWS (VMC) support
+
+## Introduction <a id="introduction"></a>
+
+[VMware Cloud™ on AWS](https://cloud.vmware.com/vmc-aws) brings VMware’s enterprise-class SDDC software to the AWS Cloud with optimized access to AWS services. Powered by VMware Cloud Foundation, VMware Cloud on AWS integrates our compute, storage and network virtualization products (VMware vSphere®, vSAN™ and NSX®) along with VMware vCenter management, optimized to run on dedicated, elastic, bare-metal AWS infrastructure.
+
+VMware Cloud on AWS provides two vSAN datastores in each SDDC cluster: WorkloadDatastore, managed by the Cloud Administrator, and vsanDatastore, managed by VMware. As a cloudadmin user, he/she does not have privilege to create volume on vsanDatastore, and only have privilege to create volume on WorkloadDatastore.
```

Comment:
`As a cloudadmin user, he/she does not have privilege to create volume on vsanDatastore` => `Cloudadmin user does not have the privilege to create volume on vsanDatastore`

Reply:
Fixed.